src/test/java/uk/gov/pay/api/service/SearchRefundsServiceTest.java

```diff
@@ -190,6 +190,33 @@ public void getAllRefundsShouldReturnCorrectFromAndToDateFromLedger() {
         assertThat(results.getLinks().getSelf().getHref(), is("http://publicapi.test.localhost/v1/refunds?from_date=2018-09-21T13%3A22%3A55Z&to_date=2018-10-23T13%3A24%3A55Z&display_size=500&page=1"));
     }
 
+    @Test
+    @PactVerification({"ledger"})
+    @Pacts(pacts = {"publicapi-ledger-search-refunds-with-from-and-to-date"})
+    public void getAllRefundsShouldReturnCorrectFromAndToDateFromLedger() {
```

Comment:
This looks like a duplicate of the test above (the renamed one) with fewer assertions.
The only difference between the new and existing pact is the order of query parameters.